src/parser.jl

```diff
@@ -42,7 +44,7 @@ struct Instances
 end
 
 # This function is used to read data of an instance from all files, and agregate into an Instance structure.
-function parser(instance_name::String, instance_type::String, path_folder::String="../data/Instances_")
+function parser(instance_name::String, instance_type::String, path_folder::String="/home/lamraoui/Projets/momh-roadef2005/data/Instances_")
```

Comment:
You have to change the path_folder default value, It should be "../data/Instances_" according to what has been proposed in #3, and according to the structure of files defined.